src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -196,6 +261,39 @@ export default Vue.extend<Data, Methods, unknown, Props>({
                         "An error occurred fetching workflow details";
                 });
         },
+        getWorkflowMetadata(){
+            api.get(`/workflows/${this.selectedWorkflowKey}/`)
+                .then(({data}) => {
+                    this.workflowMetadata = workflowRunDetailsToMetadata(data.data)
+                    this.getReportDependencies()
+                })
+                .catch((error) => {
+                    this.error = error;
+                    this.defaultMessage =
+                        "An error occurred fetching workflow details";
+                });
+        },
+        getReportDependencies() {
+            const commit = this.workflowMetadata.git_commit ? `?commit=${this.workflowMetadata.git_commit}` : '';
+            api.post(`/workflows/summary/${commit}`, {
+                reports: this.workflowMetadata.reports,
+                ref: this.workflowMetadata.git_commit
+            })
+                .then(({data}) => {
+                    const reports = [...this.workflowRunStatus.reports]
+                    this.workflowRunStatus.reports.forEach((report1, index) => {
+                        const extraData = data.data.reports.find(report2 => report2.name === report1.name)
```

Comment:
Is this going to work when the workflow contains multiple instances of the same report? Won't this always return the first one?

I missed the development of the `/workflow/summary` endpoint I think - this is the new one which distinguishes default from non-default params, right? So if you post a list of reports to it and it sorts out the parameters for you, are the response reports not guaranteed to be in the same order as the reports posted? Can't you just match tyhem up by index?